packages/clay-css/src/scss/cadmin/variables/_range.scss

```diff
@@ -168,6 +162,16 @@ $cadmin-clay-range-input: map-deep-merge(
 				right: 0,
 			),
 		),
+		focus: (
```

Comment:
@HenriquePereiraa This is working for me on cadmin. You probably need to restart the storybook / server to get the CSS to recompile. 

Something that is curious to me here is that we seem to be circumventing a merge problem in the mixin by declaring it this way.